src/components/footer.tsx

```
@@ -1,4 +1,5 @@
 import { Activity, Code } from "lucide-react"
```

ESLint

[error] 'Code' is defined but never used. Allowed unused vars must match /^_+/u.

(@typescript-eslint/no-unused-vars)